modules/update-service-mirror-release.adoc

```diff
@@ -137,31 +137,21 @@ $ oc adm release mirror -a ${LOCAL_SECRET_JSON}  \
 ----
 $ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
 ----
-... Export a local temporary repository name:
+... Take the media to the restricted network environment and upload the images to the local container registry.
 +
 [source,terminal]
 ----
-$ TEMP_REPOSITORY='<temp_repository_name>'
+$ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror "file://openshift/release:${OCP_RELEASE}*" ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} <1>
 ----
 +
-For `<temp_repository_name>`, specify the name of a temporary repository to create in your
-registry, such as `ocp4/temp`.
-... Take the media to the restricted network environment and upload the images to the temporary repository in the local container registry. By uploading the images to the temporary repository, you can then mirror them to the final repositories so that the release images are mirrored to a separate repository.
+<1> For `REMOVABLE_MEDIA_PATH`, you must use the path where you mounted the removable media.
 +
-[source,terminal]
-----
-$ oc image mirror -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror "file://openshift/release:${OCP_RELEASE}*" ${LOCAL_REGISTRY}/${TEMP_REPOSITORY} <1>
-----
++... Mirror the release image to a separate repository:
```

Comment:
The format of this item is not correct. I don't see the number in the preview. I guess it's supposed to start with `...` instead of `+...`.

Reply:
Fixed with 7ecf7c97c -> 5fa060baa, which also rebases onto master.

Reply:
[Rendered][1] view of 5fa060b has the entry numbered, so I'm marking this thread resolved.

![image](https://user-images.githubusercontent.com/209920/124839205-34658680-df3d-11eb-9d26-fd6042f62cf2.png)

[1]: https://deploy-preview-33841--osdocs.netlify.app/openshift-enterprise/latest/updating/installing-update-service?utm_source=github&utm_campaign=bot_dp